ci-script-windows.sh

```diff
@@ -56,13 +56,8 @@ execute(){
 build_darktable() {
     cd "$(dirname "$0")"
 
-    # for RawSpeed submodule
-    git submodule init
-    git submodule update
-
-    mkdir artifacts
     mkdir build && cd build
-    cmake -G "MSYS Makefiles" -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/c/projects/darktable/artifacts ../.
+    cmake -G "MSYS Makefiles" -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/c/projects/darktable/build ../.
```

Comment:
Actually, addign `bash` is wrong, it has to be `sh` as that's what the shell scripts request. And IMO it's really ugly and bad style to hard code those in the cmake files. I'd rather stick with `make` than do that.